aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -86,6 +86,9 @@ int64_t get_sub_byte_tensor_size(int64_t size_bytes, at::ScalarType t) {
     case at::ScalarType::QUInt4x2:
       new_size_bytes = std::ceil(size_bytes * 0.5);
       break;
+    case at::ScalarType::QUInt2x4:
+      new_size_bytes = std::ceil(size_bytes * 0.25);
```

Comment:
Not related to this PR but I think this code has a bug. 
for 4x2 type num bytes its celing function But it should be applied for the innermost dimension.
So if you have tensor of size say 31x31 for 4 bit you would want to allocate 16 bytes per row Thus total of 16 x 31 = 496 bytes. But if you do 31 x 31 = 961 bytes then ceil(961 * 0.5) = 481 bytes.

Since we are modifying this and adding support 2bits, can we also fix this?

Reply:
While you're at it, can you use one of the idioms for round to ceil integer division that doesn't go through floating point. If your numbers here are big the floating point inaccuracy will also be a problem.

Reply:
at::ceil_div(size_bytes, 4)

Reply:
@supriyar , @ezyang could you have another look after latest update?